modules/apps/transfer/ibc_module.go

```diff
@@ -182,6 +190,17 @@ func (im IBCModule) OnSendPacket(
 		return err
 	}
 
+	if ics20Version == types.V1 {
+		// ics20-1 only supports a single coin, so if that is the current version, we must only process a single coin.
+		if len(data.Tokens) > 1 {
+			return errorsmod.Wrapf(ibcerrors.ErrInvalidRequest, "cannot transfer multiple coins with %s", types.V1)
+		}
+
+		if len(data.Forwarding.Hops) > 1 {
```

Comment:
```suggestion
		if len(data.Forwarding.Hops) > 0 {
```
we should have a test for this

Reply:
Yeah I wanted to create a new issue with tests, bu better to do it in this PR!